includes/class-wc-comments.php

```diff
@@ -340,6 +340,46 @@ public static function get_average_rating_for_product( &$product ) {
 		return $average;
 	}
 
+	/**
+	 * Utility function for getting review counts for multiple products in one query. This is not cached.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @param array $product_ids Array of product IDs.
+	 *
+	 * @return array
+	 */
+	public static function get_review_counts_for_product_ids( $product_ids ) {
+		global $wpdb;
+
+		if ( empty( $product_ids ) ) {
+			return array();
+		}
+
+		$product_id_string = implode( "','", array_map( 'esc_sql', $product_ids ) );
+
+		$review_counts = $wpdb->get_results(
+			// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
```

Comment:
How come we're bypassing `prepare` here?

Reply:
This is because of `$product_id_string` which we were expanding for `IN` clause. I thought this is the standard way to use when doing interpolation for `IN` clause.

But after your comment, I looked in detail and see how WP core is doing it, and found this reference: https://github.com/pantheon-systems/wordpress-network/blob/master/wp-includes/class-wp-meta-query.php#L639. 
This approach still ignores phpcs, but does not bypass `prepare` so I have changed the code to use this approach in https://github.com/woocommerce/woocommerce/pull/28814/commits/9f9475a18b6a80f1b256041b4c9e56d40a45ebe8. Let me know if this makes sense.